contributing/DEVELOPING.md

```diff
@@ -284,6 +286,17 @@ you will need set the `AMP_TESTING_HOST` environment variable. (On heroku this
 is done through
 `heroku config:set AMP_TESTING_HOST=my-heroku-subdomain.herokuapp.com`)
 
+### Testing with Firebase
+For deploying and testing local AMP builds on [Firebase](https://firebase.google.com/), install firebase and initialize firebase within this directory, setting the `public` folder to `firebase` (a `firebase` folder can be generated with the command, `gulp firebase`).
+```
+npm install -g firebase-tools
+firebase login
+firebase init
+gulp firebase
```

Comment:
does gulp provide anything useful here? how about we just turn this into a normal node script

Reply:
gotcha, thats fair then if other gulp tasks might be invoked prior